TR-001

RickGutler opened the pull request.

RickGutler committed:
Criação das migrations

neste commit, foi criado as migrations para criar as tabelas no banco de dados.

RickGutler committed:
correção de um atributo

o atributo precoUnitario estava escrito precoUnitatio.

Prezotti reviewed (changes requested):
-Colocar o atributo "unsigned" nas colunas que sempre serão positivas.
-Trocar o tipo date na tabela Compra e na tabela Feira para datetime.

RickGutler committed:
troquei o que foi requisitado e atualizei nome de pastas

utilizei unsigned nos valores que nunca serão negativos, troquei os tipos de data para datetime. além disso renomeei os nomes de pastas para o padrão do projeto
